CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## 7.0.0 - 2022-04-??
+
+Are you ready to launch a [Human Rating Program](https://haystackconf.com/2019/human-judgement/)?   Quepid is now finally able to support you and your fellow human judges in gathering explicit ratings on your search results.
+
+Since the dawn of Quepid, we've suffered from an _ahem_ sub optimal rating interface for gathering human judgements.
+
+The rating interface failed most best practices for human rating, including suffering from position bias by listing documents in the order the search engine returned them, only supporting a single rating per query/doc pair, requiring lots of mouse movement and clicking.
+All that, and the UI is a combination of that required for a Relevancy Engineer with that of a Human Judge.  
+It's enough to make you weep.  
+Just to make life harder, the rating interface requires a live connection to your search engine, which often required additional technical hoops to be jumped by your decidedly untechnical subject matter experts.
+
+However that is no longer the case!
+
+![Quepid Human Rater Interface](./docs/rating-card-interface.png)
+
+This rating interface features:
+ * You can have up to three independent ratings for every query/doc pair, opening the door to interesting measurements of rating quality.
+ * A static dataset for rating that is populated from your live search engine.  Now your set of query/doc pairs won't change over time unless you want them to.
+ * Query Doc Pairs are randomly sampled, but with a bias in favour of higher positioned results over lower positioned so you can get to meaningful numbers quicker.
```

Comment:
Query Doc Pairs vs. query/doc pairs.

"query/doc pair" seems to be the spelling that is most frequently used, so I'm suggesting this to be the one used throughout the release notes.